src/app/reporting/reporting.cpp

```diff
@@ -259,15 +268,17 @@ void emberAfPluginReportingTickEventHandler(void)
             {
                 emberAfReportingPrintln("Reporting Entry Full - creating new report");
             }
-            conditionallySendReport(apsFrame->sourceEndpoint, apsFrame->clusterId);
+            conditionallySendReport(apsFrame->sourceEndpoint, apsFrame->clusterId, reportDestination, destinationEndpoint);
```

Comment:
If we have implemented `emberAfGetDestinationByBinding`, here it can be:

```
conditionallySendReport(apsFrame->sourceEndpoint, apsFrame->clusterId, emberAfGetDestinationByBinding(apsFrame), destinationEndpoint);
```